source/includes/openshift/deploy-minio-on-openshift.rst

```diff
@@ -136,7 +136,92 @@ The web console displays a widget for tracking the installation progress.
 
 Once installation completes, click :guilabel:`View Operator` to view the MinIO Operator page. 
 
-3) Open the MinIO Operator Interface
+3) Configure TLS Certificates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have installed the MinIO Operator from Red Hat OperatorHub, the installation process also configures the :openshift-docs:`OpenShift Service CA Operator <security/certificate_types_descriptions/service-ca-certificates.html>`.
+This Operator manages the the TLS certificate required to access the MinIO Operator Console and Tenants.
+It automatically renews and rotates the certificate 13 months before expiration.
+No additional action is required.
+
+For Operator installations deployed from Red Hat Marketplace or other methods, configure the :openshift-docs:`Service CA certificates <security/certificate_types_descriptions/service-ca-certificates.html>` manually.
+See the instructions in the dropdown below for details.
+
+.. dropdown:: OpenShift Service CA Certificate configuration
+
+   To manually enable the ``service-ca`` Operator to manage TLS certificates:
+
+   #. Configure the following MinIO :ref:`environment variables <minio-server-environment-variables>`:
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Environment Variable
+           - Value
+
+         * - :envvar:`MINIO_CONSOLE_TLS_ENABLE`
```

Comment:
I found this in https://github.com/minio/operator/blob/3baa9696df34b127c2fdb20306192bd44a1c86bd/docs/env-variables.md but it's not in the giant list of envvars at the moment. I think most of the envvars in that file are not.

Reply:
Appears `OPERATOR_STS_ENABLED` is not required unless you are messing with STS. Which is beyond the scope of this page. It defaults to `off`.